cmd/adsysd/client/policy.go

```diff
@@ -300,6 +319,58 @@ func (a *App) dumpCertEnrollScript() error {
 	return os.WriteFile("cert-autoenroll", []byte(script), 0600)
 }
 
+// printTicketPath prints the path to the Kerberos ccache of the given (or current) user to stdout.
+// The function is a no-op if the detect_cached_ticket setting is not enabled.
+// No error is raised if the inferred ticket is not present on disk.
+func (a *App) printTicketPath(username string) (err error) {
```

Comment:
Do you think we can have an integration test here which is setting some environment/values so that we can mock libkbr5 ticket path and asserting on this command behaviour? It seems it’s not straightfoward but if we can achieve path mocking…

Then, the e2e one would be "only" PAM + CLI ingration.